<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSBizLiveDanmaku</class>
 <widget class="QWidget" name="GBSBizLiveDanmaku">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1251</width>
     <height>931</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #01C667;
font-size:16px;
</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <widget class="QWidget" name="tabAdvanced">
    <attribute name="title">
     <string>弹幕平台设置</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>95</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>本机接口设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>380</y>
       <width>95</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>平台直播间地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>135</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF;
font-size:16px;</string>
     </property>
     <property name="text">
      <string>谷歌EXE路径</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>135</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF</string>
     </property>
     <property name="text">
      <string>服务器地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>135</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF</string>
     </property>
     <property name="text">
      <string>服务器端口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>300</y>
       <width>135</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF</string>
     </property>
     <property name="text">
      <string>服务器类型</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>430</y>
       <width>135</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF</string>
     </property>
     <property name="text">
      <string>抖音地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>480</y>
       <width>135</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF</string>
     </property>
     <property name="text">
      <string>快手地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>530</y>
       <width>135</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF</string>
     </property>
     <property name="text">
      <string>视频号地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>580</y>
       <width>135</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF</string>
     </property>
     <property name="text">
      <string>TikTok地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>630</y>
       <width>135</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF</string>
     </property>
     <property name="text">
      <string>淘宝地址</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>680</y>
       <width>135</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF</string>
     </property>
     <property name="text">
      <string>美团地址</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leChromePath">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>150</y>
       <width>581</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="leServer">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>200</y>
       <width>581</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePort">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>250</y>
       <width>581</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="leServerType">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>300</y>
       <width>581</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDouyin">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>430</y>
       <width>581</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="leKuaishou">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>480</y>
       <width>581</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="leWechatVideo">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>530</y>
       <width>581</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTikTok">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>580</y>
       <width>581</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTaobao">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>630</y>
       <width>581</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="leMeiTuan">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>680</y>
       <width>581</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="leOther">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>780</y>
       <width>581</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>780</y>
       <width>135</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF</string>
     </property>
     <property name="text">
      <string>Facebook</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePddLive">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>730</y>
       <width>581</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>730</y>
       <width>135</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF</string>
     </property>
     <property name="text">
      <string>拼多多</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbxPlat">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>160</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF</string>
     </property>
     <property name="text">
      <string>平台选择</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>30</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:5px;
color:#FFFFFF</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>弹幕回复设置</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>484</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_15">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>100</y>
       <width>484</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_16">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>484</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_17">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>160</y>
       <width>484</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> background-color: #F9F9F9;
 border: none;
border-radius: 1px;
font-size: 16px	
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>126</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>弹幕回复（文字）</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>158</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>弹幕回复（真人语音）</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>26</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:16px;</string>
     </property>
     <property name="text">
      <string>1.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>26</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size:16px
</string>
     </property>
     <property name="text">
      <string>2.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>55</y>
       <width>64</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>模糊匹配</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>55</y>
       <width>64</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>精准匹配</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAdd">
     <property name="geometry">
      <rect>
       <x>1140</x>
       <y>50</y>
       <width>24</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/gbs/images/gbs/biz/gbs-add.png);
border: none;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDel1">
     <property name="geometry">
      <rect>
       <x>1140</x>
       <y>107</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/gbs/images/gbs/biz/gbs-delete.png);
border: none;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDel2">
     <property name="geometry">
      <rect>
       <x>1140</x>
       <y>170</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/gbs/images/gbs/biz/gbs-delete.png);
border: none;
border-radius: 5px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
